lib/tokenlist.cpp

```diff
@@ -1217,14 +1217,23 @@ void TokenList::validateAst() const
             // Skip pure virtual functions
             if (Token::simpleMatch(tok->previous(), ") = 0"))
                 continue;
+            // Skip operator definitions
+            if (Token::simpleMatch(tok->previous(), "operator"))
+                continue;
             // Skip incomplete code
             if (!tok->astOperand1() && !tok->astOperand2() && !tok->astParent())
                 continue;
-            // FIXME
-            if (Token::Match(tok->previous(), "%name%") && Token::Match(tok->next(), "%name%"))
-                continue;
             // Skip lambda assignment and/or initializer
-            if (Token::Match(tok, "= {|^"))
+            if (Token::Match(tok, "= {|^|["))
+                continue;
+            // FIXME: Workaround broken AST assignment in type aliases
```

Comment:
So these FIXMEs are the conditions that should be removed after the release? Sounds good!

Reply:
Yes